### PR TITLE
Added replication2_server suite.

### PR DESCRIPTION
**Pull request by maierlars**

### CODE REVIEW

File: scripts/runFullTests.fish [hunk 1]
```diff
@@ -102,6 +102,7 @@ set ST "$ST""500,runSingleTest1 paths_server -\n"
 set ST "$ST""250,runSingleTest1 hot_backup -\n"
 # The suite doesn't have single-server tests (yet)
 #set ST "$ST""250,runSingleTest1 replication2_client -\n"
+#set ST "$ST""250,runSingleTest1 replication2_client -\n"
```
Comment (KVS85) [inline scripts/runFullTests.fish:105]:
```suggestion
#set ST "$ST""250,runSingleTest1 replication2_server -\n"
```

**Review by KVS85 — COMMENTED** [review]
LGTM.

File: scripts/runFullTests.fish [hunk 2]
```diff
@@ -217,6 +218,7 @@ set CT "$CT""250,runClusterTest1 hot_backup - --dumpAgencyOnError true\n"
 set CT "$CT""250,runClusterTest1 arangobench - --dumpAgencyOnError true\n"
 set CT "$CT""9600,runClusterTest1 chaos - --dumpAgencyOnError true --skipNightly false\n"
 set ST "$ST""250,runSingleTest1 replication2_client -\n"
+set ST "$ST""250,runSingleTest1 replication2_server -\n"
```
Comment (jsteemann) [inline scripts/runFullTests.fish:221]: Do we want to add the `--dumpAgencyOnError true` thing here as we did in https://github.com/arangodb/oskar/pull/370/files#diff-f90dcac39955f6b516159354119dfeaa12aa3b7f1751b6f32fc9fe6832c7ecd3R183?
Not sure if it is necessary, but it looks a bit inconsistent right now.

Comment (maierlars) [reply scripts/runFullTests.fish:221]:
```suggestion
set ST "$ST""250,runSingleTest1 replication2_server - --dumpAgencyOnError true\n"
```

**Review by KVS85 — APPROVED** [review]
LGTM.

**Pull request merged**